advisories/github-reviewed/2024/09/GHSA-78vg-7v27-hj67/GHSA-78vg-7v27-hj67.json

```diff
@@ -32,7 +28,7 @@
               "introduced": "0"
             },
             {
-              "fixed": "6.0.0"
```

Comment:
5.2.6, 6.0.0

Reply:
fix was introduced in 6.0.0 then back ported to 5.2.6 (already updated in https://github.com/DamienHarper/auditor-bundle/security/advisories/GHSA-78vg-7v27-hj67)